Formula/p/python-boto3.rb

```diff
@@ -15,6 +15,8 @@ class PythonBoto3 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "68ed27c847e1075678e09d39f8b0da39354d28af2992de0120436fbcf9563006"
   end
 
+  deprecate! date: "2024-02-19", because: "is not allowed in homebrew-core"
```

Comment:
Could remove instead, though there are some installs in https://formulae.brew.sh/analytics/install-on-request/365d/ (720).

Also, no idea about reason. None of pre-existing reasons make sense so just aligned reason with upcoming documentation.

Reply:
I vote for outright removal for these.

Reply:
Yeah, I think we can just remove these.

Reply:
n00b question: do we know of specific reasons why someone would opt to explicitly install a `python-*` formula that isn't providing some CLI tool? It seems to me like if someone is bringing in a dependency they would be pretty much always using a virtualenv or environment manager. It seems to me like some installs could just be from folks who are have issues getting a working Python environment and are trying whatever seems like it might work?